test/services/BaseRepositoryTest.java

```diff
@@ -527,7 +528,7 @@ public void testSearchBySubjectClassification() throws IOException, InterruptedE
   @Test
   public void testSearchByEducationClassification() throws IOException, InterruptedException {
```

Comment:
Don't know why this one is failing on Travis, it runs fine in isolation locally: https://travis-ci.org/hbz/oerworldmap/builds/196599573#L1417